modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -58,10 +58,42 @@ Card Border:: Specifies the card border thickness.
 Card Border Radius:: Specifies the card border radius. A larger radius makes the cards look more rounded.
 Card Shadow:: Specifies the shadow beneath each card.
 Message If Empty:: Shows a message if the carousel has no APIs. Default is `There are no APIs available right now.`
-Filter Condition (Advanced):: Limits the APIs shown in the carousel to those matching a standard Salesforce Object Query Language (SOQL) search. SOQL searches use package names, field names such as "API name" or "API description," and comparison of strings to the values of those fields.
+Filter Condition (Advanced):: Causes the carousel to show only APIs that match the given search condition, which uses the syntax described in the following section.
```

Comment:
Beware ever referring to another section, because things move around and change. Better to name the title or refer to it generally: "...described in the filter conditions documentation."

Reply:
Good point. Updated this to name the title and link to the section.